site/source/docs/porting/multimedia_and_graphics/OpenGL-support.rst

```diff
@@ -40,6 +42,8 @@ This allows you to use functions `glDrawArrays <https://www.opengl.org/sdk/docs/
 
 To enable *OpenGL ES 2.0 emulation*, specify the :ref:`emcc <emcc-s-option-value>` option ``-s FULL_ES2=1`` when linking the final executable (.js/.html) of the project.
 
+To enable *OpenGL ES 3.0 emulation*, specify the :ref:`emcc <emcc-s-option-value>` option ``-s FULL_ES3=1`` when linking the final executable (.js/.html) of the project. This adds emulation for mapping memory blocks to client side memory.
```

Comment:
The flags `-s FULL_ES2=1` and `-s FULL_ES3=1` are orthogonal that they emulate different features, and it looks like `-s FULL_ES3=1` does not imply or require `-s FULL_ES2=1`. Added a clarification about that in the docs. (I don't think it would be bad if we did make `-s FULL_ES3=1` to imply `-s FULL_ES2=1`, but at present no reason to go changing towards that either)